Android.mk

```diff
@@ -110,6 +113,7 @@ LOCAL_SRC_FILES := \
     src/utils/event_emitter.cpp \
     src/utils/hex.cpp \
     src/utils/strcpy_utils.cpp \
+    src/utils/system_utils.cpp \
```

Comment:
To prevent such issues from happening again, android-check should be enhanced to turn 1.2 features on.